packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -35,17 +35,11 @@ library TaikoData {
         // The size of the ETH deposit ring buffer.
         uint256 ethDepositRingBufferSize;
         // The minimum number of ETH deposits allowed per block.
-        uint64 ethDepositMinCountPerBlock;
+        uint64 ethDepositsPerBlock;
```

Comment:
I think we lose backward compatiblity with Hekla with the removal of these:
`ethDepositMaxAmount`, `ethDepositGas`, `ethDepositMaxFee`

because the `blockSyncThreshold` position within the struct slots will be changed, and we use that in LibVerifying.sol:

`if (_lastVerifiedBlockId > lastSyncedBlock + _config.blockSyncThreshold) `

Reply:
If all smart contracts are updated should be good I guess because will point to the correct location?

Reply:
Ah ok, since `memory` object, right !

Reply:
@Brechtpd, @adaki2004 do you prefer the entire EthDeposit feature is removed instead? Now we have our Bridge and it seems nobody used the EthDeposit feature, and the current implementation of this feature also assume the existence of the Bridge.